src/03-control-flow.js

```diff
@@ -86,7 +114,15 @@ function positiveNegativeOrZero(a) {}
  *  aroundTheWorldGreeting();
  *  //> "Hello World"
  */
-function aroundTheWorldGreeting(language) {}
+function aroundTheWorldGreeting(language) {
```

Comment:
We could have added a default value of english to our function parameter `function aroundTheWorldGreeting(language="english")` so we can more explicitly describe our intentions here.